modules/preferences/main/index.coffee

```diff
@@ -284,10 +295,10 @@ class Preferences extends hx.EventEmitter
   supportedTimezones: option 'supportedTimezones'
   timezoneOffsetLookup: option 'timezoneOffsetLookup'
 
-  applyTimezoneOffset: (date, offset) ->
```

Comment:
Looking at this, I think the new code you've replaced it with does exactly the same as before. Is the idea just to split it up a bit to make it clearer?

If thats the case, could we make `handleTimeZoneOffsetDifferences` a function rather than a method?